test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -283,8 +283,8 @@ public static IEnumerable<object[]> MessageSizesData
                     logger.LogInformation("Sent message", bytes.Length);
 
                     logger.LogInformation("Receiving message");
-                    // No timeout here because it can take a while to receive all the bytes
-                    var receivedData = await receiveTcs.Task;
+                    // Big timeout here because it can take a while to receive all the bytes
```

Comment:
👍 👍 👍

Reply:
This just means there’s a hang still...

Reply:
I just changed it so that you know there is a problem after 1 minute instead of hanging the CI server until it kills the entire build.

Reply:
yeap